mintpy/ifgram_inversion.py

```diff
@@ -1084,7 +1086,10 @@ def ifgram_inversion(ifgram_file='ifgramStack.h5', inps=None):
         # at once (other user's jobs gained higher priority in the general at that point)
         NUM_WORKERS = inps.numWorker
         # FA: the following command starts the jobs
-        cluster = cl.get_cluster(type=inps.cluster, walltime=inps.walltime)
+        if inps.clusterConfig == "None":
```

Comment:
I recommend changing the default value to `no` because: `None` from default parser in ifgram_inversion.py and `"None"` from template file are not consistent with each other, there should be a bug here.

Reply:
Yes. I was wondering how to do handle this case. I didn't like having string None, but I wasn't sure how else to deal with it. "no" makes sense as a default value.